LISAv2-Framework.psm1

```diff
@@ -186,7 +187,7 @@ function Start-LISAv2 {
 			Write-LogInfo "Test $global:testId finished"
 
 			# Output text summary
-			$plainTextSummary = $testController.TestSummary.GetPlainTextSummary($OsVHD, $ARMImageName, $OverrideVMSize)
+			$plainTextSummary = $testController.TestSummary.GetPlainTextSummary($OsVHD, $testController.ARMImageName, $OverrideVMSize)
```

Comment:
Why we dont use ArmImageName which is passed as an argument ? Is it now unused parameter ?

Reply:
The ARMImageName is raw value here,  we changed to use testController.ARMImageName (actually AzureController) for  a well formatted log and report.  The AzureController.ARMImageName is a well treated value, e.g., it trimed multiple blank spaces between publisher, offer and sku, version.   Using neat and consistent format.